.github/workflows/update-ts-version-tags.yml

```diff
@@ -15,19 +15,24 @@ on:
     # https://crontab.guru/#5_8_*_*_1
     - cron: "5 8 * * 1"
 
+env:
+  CI: true
+  PNPM_CACHE_FOLDER: .pnpm-store
+
 jobs:
   publish:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
     - uses: actions/setup-node@v2
       with:
-        node-version: '14'
+        node-version: 'lts/*'
+    - run: npm install -g pnpm
+    - run: pnpm config set store-dir $PNPM_CACHE_FOLD
```

Comment:
FWIW there's also an action-y way to do this which may be faster: https://github.com/jakebailey/esbuild-playground/blob/2149487e8726b41ecac8ae95e2c50706b9047dca/.github/workflows/gh-pages.yml#L37

Or just setting `cache: pnpm` on the setup-node action, though I'm pretty sure that doesn't work because pnpm's not installed in time.

Reply:
this also came from pnpm's guide 🤷

Reply:
weird, must be a different guide from https://github.com/pnpm/action-setup

Reply:
https://pnpm.io/using-changesets